phpcs.xml.dist

```diff
@@ -1,62 +1,81 @@
 <?xml version="1.0"?>
 <ruleset name="WP Job Manager">
-	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
-	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
-
-	<!-- Set a description for this ruleset. -->
 	<description>A custom set of code standard rules to check for WordPress themes and plugins.</description>
 
-	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs,vendor/phpcompatibility/php-compatibility" />
+	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs,vendor/phpcompatibility/phpcompatibility-paragonie,vendor/phpcompatibility/php-compatibility,vendor/phpcompatibility/phpcompatibility-wp,vendor/sirbrillig/phpcs-variable-analysis" />
+
+	<arg value="s"/>
+	<arg name="colors"/>
+
+	<arg name="extensions" value="php"/>
+	<file>.</file>
+
+	<exclude-pattern>node_modules/</exclude-pattern>
+	<exclude-pattern>vendor/</exclude-pattern>
+	<exclude-pattern>tests/</exclude-pattern>
+	<exclude-pattern>tmp/</exclude-pattern>
 
-	<exclude-pattern>*/node_modules/*</exclude-pattern>
-	<exclude-pattern>*/vendor/*</exclude-pattern>
-	<exclude-pattern>*/lib/*</exclude-pattern>
-	<exclude-pattern>*/tests/bin/*</exclude-pattern>
-	<!-- Exclude templates for now -->
-	<exclude-pattern>*/templates/*</exclude-pattern>
+	<!-- Temporarily exclude templates and lib for now -->
+	<exclude-pattern>templates/</exclude-pattern>
+	<exclude-pattern>lib/</exclude-pattern>
 
 	<!-- Configs -->
 	<config name="minimum_supported_wp_version" value="4.7" />
 	<config name="testVersion" value="5.2-"/>
 
 	<!-- Rules -->
-	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
-		<exclude-pattern>includes/**/abstract-*.php</exclude-pattern>
-		<exclude-pattern>tests/*</exclude-pattern>
+	<rule ref="PHPCompatibilityWP"/>
+	<rule ref="WordPress-Core" />
```

Comment:
No need to include `WordPress-Core`, when you have `Wordpress-Extra`, as Extra already includes Core.